### PR TITLE
Bug 1152666 - [System] fix jshint errors/warnings for telephony related ...

### PR DESCRIPTION
**Pull request by huchengtw-moz**
...files

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=660b6574e3d52618860e8e06087028cb9ec3a747

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=83b8c708ecd1106b998de68311ebb6854a0edf9d

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=60a76a8c148cdd3f24b53ea8b5f729544dfb5b10

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=13dc3ecc45e7e27da060be4816b3d32e955bd75e

File: apps/system/js/icc_events.js [hunk 1]
```diff
@@ -273,10 +274,10 @@ var icc_events = {
       case icc._iccManager.STK_EVENT_TYPE_NETWORK_SEARCH_MODE_CHANGED:
       case icc._iccManager.STK_EVENT_TYPE_BROWSING_STATUS:
       case icc._iccManager.STK_EVENT_TYPE_FRAMES_INFORMATION_CHANGED:
-        DUMP('icc_events_register - TODO event: ', eventList[evt]);
+        DUMP('icc_events_register - TODO event: ', evtId);
         break;
       }
-    }
+    }.bind(this));
```
Comment (cctuan) [inline apps/system/js/icc_events.js]: }, this);

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eba2233d73780903bb8549796f3a47209fa37cda

**Pull request merged**